assets/javascripts/discourse/components/fields/da-custom-field.gjs

```diff
@@ -0,0 +1,46 @@
+import BaseField from "./da-base-field";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import DAFieldLabel from "./da-field-label";
+import DAFieldDescription from "./da-field-description";
+import MultiSelect from "select-kit/components/multi-select";
+import { hash } from "@ember/helper";
+import { inject as service } from "@ember/service";
+
+export default class GroupField extends BaseField {
+  <template>
+    <section class="field group-field">
+      <div class="control-group">
+        <DAFieldLabel @label={{@label}} @field={{@field}} />
+
+        <div class="controls">
+          <MultiSelect
+            @content={{this.allCustomFields}}
+            @value={{@field.metadata.value}}
+            @labelProperty="name"
+            @onChange={{this.setCustomField}}
+            @options={{hash maximum=1 disabled=@field.isDisabled}}
+          />
+
+          <DAFieldDescription @description={{@description}} />
+        </div>
+      </div>
+    </section>
+  </template>
+
+  @service store;
+  @tracked allCustomFields = [];
+
+  constructor() {
+    super(...arguments);
+
+    this.store.findAll("user-field").then((fields) => {
+      this.allCustomFields = fields.content
+    })
+  }
+
+  @action
+  setCustomField(ids) {
```

Comment:
I think if you use `ComboBox` too you won't have to do this `ids` thing, should be able to directly `mut` the value in the template